source/code/core/core/public/ice/os/windows.hxx

```diff
@@ -10,7 +11,94 @@
 namespace ice::win32
 {
 
-    using ModuleHandle = HMODULE;
+    template<typename HandleType>
+    class SafeHandle
```

Comment:
This class is kinda helpfull, but should be replaced / improved later.  Create a tech-debt issue after closing #95.

Reply:
Created #112